### PR TITLE
feat: cutdown tokio features

### PR DESCRIPTION
**Pull request by OlshaMB**
Cutdown tokio to minimal setup. Cuts down binary from 24mb to 16mb

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ serde = { version = "1.0.163", features = ["derive"] }
 teletypewriter = { path = "teletypewriter" }
 sugarloaf = { path = "sugarloaf" }
 corcovado = { path = "corcovado" }
-tokio = { version = "1.26.0", features = ["full"] }
+tokio = { version = "1.26.0", features = ["rt", "rt-multi-thread", "macros"] }
```
Comment (raphamorim) [inline Cargo.toml:31]: would you mind update tokio as well to "1.29.1"?

**Comment by OlshaMB** [comment]
Done

**Comment by OlshaMB** [comment]
Why you didn't merge?

**Comment by raphamorim** [comment]
Thanks @OlshaMB 🥇

**Pull request merged**